NtupleProducer/python/scripts/objMultiplicityPlot.py

```diff
@@ -13,6 +13,7 @@
 parser = OptionParser("%(prog) infile [ src [ dst ] ]")
 parser.add_option("--cl", type=float, dest="cl", default=0, help="Compute number to avoid truncations at this CL")
 parser.add_option("-p", type="string", dest="particles", action="append", default=[], help="objects to count")
+parser.add_option("-d", dest="detector", choices=["l1pfProducerBarrel","l1pfProducerHF","l1pfProducerHGCal","l1pfProducerHGCalNoTK"], default="l1pfProducerBarrel", help="choice of detector: l1pfProducerBarrel, l1pfProducerHGCal, l1pfProducerHGCalNoTK, l1pfProducerHF")
```

Comment:
I would remove "l1pfProducer" from all, and just add it later when getting the variable